Update basic-sql-operations.md (#3401)
pingcap/docs: basic-sql-operations.md

SHOW INDEX from person;
```

To delete an index, use the `DROP INDEX` or `ALTER TABLE` statement. `DROP INDEX` can be nested in `ALTER TABLE`:

{{< copyable "sql" >}}

```sql
DROP INDEX person_id ON person;
```

{{< copyable "sql" >}}

```sql
ALTER TABLE person DROP INDEX person_unique_id;
```

> **Note:**
> 

> > DDL operations are not transactions. You don't need to correspond COMMIT statements when executing DDL.

Reviewer: ```suggestion
> DDL operations are not transactions. You don't need to run a `COMMIT` statement when executing DDL operations.
```

INSERT INTO person VALUES("1","tom","20170912");
```

To insert a record containing data of some fields into a table, use the `INSERT` statement:

{{< copyable "sql" >}}

```sql
INSERT INTO person(id,name) VALUES("2","bob");
```

To update some fields of a record in a table, use the `UPDATE` statement:

{{< copyable "sql" >}}

```sql
UPDATE person SET birthday='20180808' WHERE id=2;
```

To delete the data in a table, use the `DELETE` statement:

{{< copyable "sql" >}}

```sql
DELETE FROM person WHERE id=2;
```

> **Note:**
> 
> The `UPDATE` and `DELETE` statements without the `WHERE` clause as a filter operate on the entire table.

DQL is used to retrieve the desired data rows from a table or multiple tables.

## Query data

To view the data in a table, use the `SELECT` statement:

{{< copyable "sql" >}}

```sql
SELECT * FROM person;
```

> Add the column name to be queried after the `SELECT` statement:

Reviewer: ```suggestion
To query a specific column, add the column name after the `SELECT` keyword:
```
Please update other descriptions as well.
